src/test/resources/feature/BDDscenarioForWishList.feature

```diff
@@ -0,0 +1,46 @@
+@WishList
+Feature: Wish List;
+
+  Background:
+    Given user is on http://automationpractice.com/ page
+
+
+  @CreateWishList
+  Scenario Outline: Create Wish List
+    Given user is logged in
+    And user is on MY WISHLISTS page
+    And user type on name field "<name>"
+    When user click on Save button
+    Then Wishlist is created
+
+    Examples:
+      | name         |
+      | My favourite |
+      | 1235         |
+
+
+  @AddToWishList
+  Scenario: Add to wishlist/see the item
+    Given user is logged in
+    And user on main page
+    And user pointed mouse on item
+    And user click on More button
+    And user is redirected on item page
+    When user click on Add to wish list button
+    And user click on profile button
+    And user is redirected on profile page
+    And user click on MY WISHLISTS button
+    And user is redirected on MY WISHLISTS page
```

Comment:
Same here, you can replace with one single step **user navigates to My Wishlists page** there is no need to be so specific. And you can handle all the logic in StepDef

Reply:
done